gbasis/contractions.py

```diff
@@ -28,8 +28,8 @@ class GeneralizedContractionShell:
 
     .. math::
 
-        \phi_{\vec{a}, \vec{R}_A} (\mathbf{r}) =
-        N_{\phi} (\vec{a}, \vec{R}_A) \sum_i d_i g_i (\vec{r} | \vec{a}, \vec{R}_A)
+        \phi_{\vec{a}, \vec{R}_A, \mathbf{d}, \boldsymbol{\alpha}} (\mathbf{r}) =
+        N_{\phi} (\vec{a}, \vec{R}_A, \mathbf{d}, \boldsymbol{\alpha}) \sum_i d_i g_i (\vec{r} | \vec{a}, \vec{R}_A)
```

Comment:
1. Both $\mathbf{r}$ and $\vec{a}$ are used in one euqation. Can you make everything $\mathbf{r}$, $\mathbf{R}$, and $\mathbf{a}$ in all equations?
2. Instead of using subscripts for $\phi$, write it in the same style as $g_i$?

Reply:
Addressed here 7eb917b11d3487f2a3e825c891d10b4fc9b3b0d5